### PR TITLE
dx: add `pilot-mainnet` script

### PR DESCRIPTION
**Pull request by tomholford**
also, address linter issue

### CODE REVIEW

File: src/Bridge.js [hunk 1]
```diff
@@ -21,9 +21,9 @@ import GlobalErrorBoundary from 'components/GlobalErrorBoundary';
 
 const INITIAL_NETWORK_TYPE = isRopsten
   ? NETWORK_TYPES.ROPSTEN
-  : isDevelopment
-  ? NETWORK_TYPES.LOCAL
-  : NETWORK_TYPES.MAINNET;
+  : isMainnet
+  ? NETWORK_TYPES.MAINNET
+  : NETWORK_TYPES.LOCAL;
```
Comment (Fang-) [inline src/Bridge.js:26]: I don't think we want to disregard the `isDevelopment` flag here, right? Default behavior for production builds should still be mainnet.

```suggestion
  : (isDevelopment && !isMainnet)
  ? NETWORK_TYPES.LOCAL
  : NETWORK_TYPES.MAINNET;
```

Comment (tomholford) [reply src/Bridge.js:26]: Good catch, LGTM

**Review by Fang- — APPROVED** [review]
Nice! Thank you.

**Comment by ngzax** [comment]
FTR, I have confirmed that this works for me. Thanks!

**Pull request merged**